tests/kuttl/common/keystoneapi_deploy.yaml

```diff
@@ -0,0 +1 @@
+../../../config/samples/keystone_v1beta1_keystoneapi.yaml
```

Comment:
interesting... so we don't need to issue any oc commands... like kuttl will just apply the crd we have defined there? https://github.com/openstack-k8s-operators/keystone-operator/blob/b9019b4533f4049d41677329134d6dddb0ab0a63/config/samples/keystone_v1beta1_keystoneapi.yaml#L1

Reply:
Exactly, if we pass a yaml file with kubernetes resources, kuttl will apply them to the cluster